app/api/domains/api.py

```diff
@@ -145,14 +145,14 @@ async def api_get_player_info(
     if username:
         user_info = await app.state.services.database.fetch_one(
             "SELECT id, name, safe_name, "
-            "priv, country, silence_end "
+            "priv, clan_id, country, silence_end "
```

Comment:
for what is even silence_end put here?

Reply:
on osu/ripple frontend u can see how long someone is silenced for, so i thought i'd include it, public enough data